command/create/create.js

```diff
@@ -12,12 +12,16 @@ const info = {
 const action = async (message, args) => {
   const name = args[0];
   if (!name) {
-    message.channel.send('Error: No name provided');
+    const dm = await message.author.createDM();
```

Comment:
Possible to merge this dm and the one below ?

Reply:
Yes I did it